pkg/noun/v2/manage.c

```diff
@@ -0,0 +1,177 @@
+/// @file
+
+#include "pkg/noun/v2/manage.h"
+
+#include "pkg/noun/v2/allocate.h"
+#include "pkg/noun/v2/hashtable.h"
+#include "pkg/noun/v2/jets.h"
+#include "pkg/noun/v2/nock.h"
+#include "pkg/noun/v2/options.h"
+#include "pkg/noun/vortex.h"
+#include "pkg/noun/v2/vortex.h"
+
+/* _cm_pack_rewrite(): trace through arena, rewriting pointers.
+ *                     XX need to version; dynamic scope insanity!
+*/
+static void
+_cm_pack_rewrite(void)
+{
+  //  XX fix u3a_v2_rewrite* to support south roads
+  //
+  u3_assert( &(u3H_v2->rod_u) == u3R_v2 );
+
+  u3v_v2_rewrite_compact();
+  u3j_v2_rewrite_compact();
+  u3n_v2_rewrite_compact();
+  u3a_v2_rewrite_compact();
```

Comment:
Technically, all these rewrite_compact functions are v1->v2 migration functions. They're *not* the original rewrite_compact functions as they were in v2 (for pack). We might want to rename them, or at least add comments.